src/sidebar/components/test/annotation-share-control-test.js

```diff
@@ -204,6 +206,15 @@ describe('AnnotationShareControl', () => {
     );
   });
 
+  it("doesn't focuses the share-URI input when opened on iOS", () => {
+    fakeIsIOS.returns(true);
+    const wrapper = createComponent();
+    openElement(wrapper);
+    wrapper.update();
+
+    assert.isNull(document.activeElement.getAttribute('aria-label'));
```

Comment:
The use of `aria-label` is a slightly indirect way of detecting the focused element, although it is consistent with the existing test. In this case I would recommend to extract a helper to provide a more obvious meaning:

```
assert.isTrue(isLinkInputFocused());
```

Where `isLinkInputFocused` (or similar) is a helper that encapsulates the details of checking whether the input field has focus.

Reply:
👍🏼